src/index.js

```diff
@@ -5,13 +5,13 @@ import Config from "./config";
 import {getUserFromJwt} from "./lib/identityActions";
 import "./index.css";
 import registerServiceWorker from "./registerServiceWorker";
-import OneGraphApolloClient from "onegraph-apollo-client";
+import ApolloClient from "apollo-boost";
 import {ApolloProvider} from "react-apollo";
 import api from "./lib/apiGraphQL";
 import {getAppVersion} from "./lib/appVersion";
 import {validateToken} from "./lib/validateToken";
 
-const apolloClient = new OneGraphApolloClient({
+const apolloClient = new ApolloClient({
```

Comment:
@sgrove do you have thoughts here. In order to leverage React v0.17 we are going with ApolloCliene over the OneGraphApolloClient.

Reply:
What is it about `OneGraphApolloClient` that doesn't work with React v0.17? We can fix that in the onegraph-apollo-client package.

It's just a thin wrapper over `@apollo/client` that sets up auth and sets the correct path. It also makes it easier to set up subscriptions, but I'm not sure if you're using that.

If you do switch to plain ApolloClient, you'll want to make sure to set up the url and the auth, similar to what we do here: https://github.com/OneGraph/onegraph-client/blob/master/packages/onegraph-apollo-client/src/OneGraphApolloClient.js#L25

Reply:
had BC updating to react17, didn't want to switch dependency resolution for reasons unrelated, so just followed documentation here: https://www.npmjs.com/package/onegraph-apollo-client which led me here: https://www.onegraph.com/docs/front_end_apollo.html - I'm fine with keeping what is but the docs definitely confused me, not being focused on onegraph/apollo other than having the frontend working. still confused on what should be updated or reverted :D